zerver/middleware.py

```diff
@@ -271,21 +288,25 @@ def process_exception(self, request, exception):
 
 class TagRequests(object):
     def process_view(self, request, view_func, args, kwargs):
+        # type: (HttpRequest, Callable[..., None], List[str], Dict[str, Any]) -> bool
```

Comment:
same comment on this view_func as above.

What makes you think this returns a bool?